OpenRA.Mods.Common/Traits/Harvester.cs

```diff
@@ -74,6 +71,9 @@ public class HarvesterInfo : ITraitInfo, Requires<MobileInfo>
 		[Desc("The pathfinding cost penalty applied for each harvester waiting to unload at a refinery.")]
 		public readonly int UnloadQueueCostModifier = 12;
 
+		[Desc("Does the unit queue harvesting runs instead of individual harvest actions?")]
+		public readonly bool QueueRuns = true;
```

Comment:
If that is the consensus, then so be it. Changed the default.

I do think that per-run better facilitates tactical play (e.g. queueing a single long-range harvesting run to get some blue tiberium before resuming on a closer patch; queueing a move order after a harvesting run to avoid an enemy troop concentration), while per-cell mainly facilitates harvester nannying. But, I don't want to get involved in that kind of discussion either. I'm content with just keeping it a modding option.